src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -367,6 +367,7 @@ export const ReviewRoute: FC<ReviewProps> = props => {
         currentStep="Review"
         steps={order.mode === "OFFER" ? offerFlowSteps : buyNowFlowSteps}
       />
+      <Spacer mb={4} />
```

Comment:
**question**: do we know why payment route is wrapped by a Box (above in 365) while other routes are not?

Reply:
ah okay, thank you for figuring it out!